src/functions/retryProcessing/framework/handler.ts

```diff
@@ -1,8 +1,20 @@
-import { APIGatewayEvent, Context } from 'aws-lambda';
+import { Context, ScheduledEvent } from 'aws-lambda';
 import createResponse from '../../../common/application/utils/createResponse';
 import { HttpStatus } from '../../../common/application/api/HttpStatus';
 import Response from '../../../common/application/api/Response';
+import { RetryProcessor } from '../application/RetryProcessor';
+import { bootstrapConfig } from '../../../../src/common/framework/config/config';
+import { IRetryProcessor } from '../application/IRetryProcessor';
+import { error } from '@dvsa/mes-microservice-common/application/utils/logger';
 
-export async function handler(event: APIGatewayEvent, fnCtx: Context): Promise<Response> {
+export async function handler(event: ScheduledEvent, fnCtx: Context): Promise<Response> {
+  await bootstrapConfig();
+  const retryProcessor: IRetryProcessor = new RetryProcessor();
+  try {
+    await retryProcessor.processRetries();
```

Comment:
We would like to continue processing the rest of the queries even though one of them failed (got rollbacked)

Reply:
Removed the throw on each each in favour of logging.